gamestonk_terminal/cryptocurrency/coinpaprika/README.md

```diff
@@ -0,0 +1,321 @@
+
+````
+usage: coins [-s --skip] [-t --top] [-l --letter] [-k --key]
+````
+
+Display all available coins in coinpaprika. You can search with pagination mechanism with `coins --skip [num] --top [num]` or you can just search by letter like
+`coins --letter d --key name --top 10` it will display top 10 matches for coins which name starts with letter `d` and search will be done in column `name`
+
+* -s/--skip - number of records to skip.  Default is 0. There are thousands of coins, so there is mechanism to paginate through them.
+e.q if you want to see only records from 500-750 you should use `coins --skip 500 --top 250` or if you want to see 1200-1880 you should use `coins --skip 1200 --top 680`.
+* -t/--top - number of news to display. One page of news contains 25 news, so to get 250 news script needs to scrape 10 pages (it can take some time). Default 100. E.g `news --top 150`
+* -l/--letter - first letters of coin by which you want to search
+* -k/--key - search key. With this parameter you can specify in which column you would like to search. Choose on from `name, id, symbol`
+
+## info <a name="info"></a>
+
+````
+usage: info [-t --top] [-s --sort] [--descend]
+````
+* --vs The currency to look against. Defaults to "USD". Available options are: [BTC,ETH,USD,EUR,PLN,KRW,GBP,CAD,JPY,RUB,TRY,NZD,AUD,CHF,UAH,HKD,SGD,NGN,PHP,MXN,BRL,THB,CLP,CNY,CZK,DKK,HUF,IDR,ILS,INR,MYR,NOK,PKR,SEK,TWD,ZAR,VND,BOB,COP,PEN,ARS,ISK]
+* -t/--top - number of coins to display. To display top 10 coins: `info --top 10 --sort rank`
+* -s/--sort - sort by given column. You can chose on from `rank, name, symbol, price, volume_24h, circulating_supply, total_supply, max_supply, ath_price,market_cap, beta_value`.
+* --descend - flag to sort in descending order (lowest first)
+
+Show basic coin information for all coins from CoinPaprika
+
+## markets <a name="markets"></a>
+
+````
+usage: markets [--vs] [-t --top] [-s --sort] [--descend]
+````
+* --vs The currency to look against. Defaults to "USD". Available options are: [BTC,ETH,USD,EUR,PLN,KRW,GBP,CAD,JPY,RUB,TRY,NZD,AUD,CHF,UAH,HKD,SGD,NGN,PHP,MXN,BRL,THB,CLP,CNY,CZK,DKK,HUF,IDR,ILS,INR,MYR,NOK,PKR,SEK,TWD,ZAR,VND,BOB,COP,PEN,ARS,ISK]
+* -t/--top - number of coins to display. To display top 10 coins: `markets --top 10 --sort rank`
+* -s/--sort - sort by given column. You can chose on from `rank, name, symbol, price, volume_24h, mcap_change_24h, pct_change_1h, pct_change_24h, ath_price, pct_from_ath`.
+* --descend - flag to sort in descending order (lowest first)
+
+Show market related (price, supply, volume) coin information for all coins on CoinPaprika
+
+
+## exchanges <a name="exchanges"></a>
+
+````
+usage: exchanges [--vs] [-t --top] [-s --sort] [--descend]
+````
+* --vs The currency to look against. Defaults to "USD". Available options are: [BTC,ETH,USD,EUR,PLN,KRW,GBP,CAD,JPY,RUB,TRY,NZD,AUD,CHF,UAH,HKD,SGD,NGN,PHP,MXN,BRL,THB,CLP,CNY,CZK,DKK,HUF,IDR,ILS,INR,MYR,NOK,PKR,SEK,TWD,ZAR,VND,BOB,COP,PEN,ARS,ISK]
```

Comment:
this is reaaaalllyyy nitpicky. On the remaining READMES we usually do:

* arg : Comment. Default: XX

Note the ":" instead of "-". Not important though :).